### PR TITLE
repair: correct misspelling of "corespondent"

### PR DESCRIPTION
**Pull request by tchaikov**
replace "corespondent" with "corresponding" in a logging message.

---

it's a cleanup, hence no need to backport.

### CODE REVIEW

**Comment by tchaikov** [comment]
@scylladb/scylla-maint hello maintainers, could you help review this change?

**Comment by regevran** [comment]
This is strange, both spellings are correct, but my English (and the time I want to spend on this) is not good enough to understand the differences.
Nor [this explanation](https://rephrasely.com/usage/corespondent-vs-correspondent) helped :confused:

**Comment by tchaikov** [comment]
> This is strange, both spellings are correct, but my English (and the time I want to spend on this) is not good enough to understand the differences. Nor [this explanation](https://rephrasely.com/usage/corespondent-vs-correspondent) helped 😕

probably this helps ?

- https://www.merriam-webster.com/dictionary/corespondent
- https://www.merriam-webster.com/dictionary/correspondent

**Comment by nyh** [comment]
> This is strange, both spellings are correct, but my English (and the time I want to spend on this) is not good enough to understand the differences. Nor [this explanation](https://rephrasely.com/usage/corespondent-vs-correspondent) helped 😕

Correspondent, with the two Rs, is the only relevant correct spelling.

The dictionaries also have "corespondent" with one R as a valid word, giving some stupid legal sense - in law, "respondent"  is similar to defendant, specifically in divorce court (the plaintiff in a divorce court is called the "petitioner"). So a "co-respondent" is a second person being blamed for the divorce, e.g., someone who participated in the adultery. This is a really fringe use case of this word and I doubt it has much use in practice - I suspect it is much more common as a misspelling of "correspondent". In fact, ironically, if you look at https://www.merriam-webster.com/dictionary/corespondent the only example sentence is a **misspelling** from 2019, and not the legal sense that the dictionary claims it is giving an example of.

**Comment by nyh** [comment]
> This is strange, both spellings are correct, but my English (and the time I want to spend on this) is not good enough to understand the differences. Nor [this explanation](https://rephrasely.com/usage/corespondent-vs-correspondent) helped 😕

That "explanation" is pure garbage probably written by a really stupid AI.

**Comment by regevran** [comment]
Thanks @nyh - indeed I first looked at Merriam and couldn't find my way through.
Now it is much clearer!

**Review by nyh — CHANGES_REQUESTED** [review]
I got caught up in the interesting question of whether corespondent or correspondent are the right spelling, but both aren't really right. The word "corresponding" is much more common as an adjective than "correspondent". I don't think I ever heard "correspondent" used as an adjective.


Again, unabridged dictionaries will tell you "correspondent" as an adjective exists, but normal people don't use it. See for example https://forum.wordreference.com/threads/corresponding-correspondent.1952217/ and https://www.italki.com/en/post/question-468036 and https://textranch.com/c/correspondent-or-corresponding/ These are normal English speakers, not unabridge-dictionary-reading AIs ;-)

Or improve this string even further... To be honest I wouldn't even know what "Provided host ip has no corresponding host id" even refers to... Where is this ip-id correspondence coming from? Does this sentence just mean that given IP address is not a Scylla node?

**Comment by nyh** [comment]
> That "explanation" is pure garbage probably written by a really stupid AI.

By the way, turns not ChatGPT is a non-stupid AI. I asked it:
> Which of the two spellings "correspondent" or "corespondent" is correct? Please explain which one I should use as an adjective to indicate that one thing corresponds to another thing.

And it told me: 

> The correct spelling for the adjective meaning "corresponding" is "correspondent."
> Explanation:
>
>    "Correspondent": This is the correct term when you want to describe something that corresponds to or is in agreement with something else. It can also be a noun meaning someone who communicates or reports (e.g., a journalist).
>
>    Example as an adjective:
>        The two datasets are correspondent in their findings.
>
>    "Corespondent": This is a completely different word and refers to a person named in legal proceedings, often in divorce cases, as someone alleged to have committed adultery with the respondent. It has no meaning related to "corresponding."
>
> For your intended usage, always use "correspondent."

**Comment by tchaikov** [comment]
v2:

- replaced "corresponding" in v1 with "corresponding" so it is easier to digest.

**Review by nyh — APPROVED** [review]

**Comment by tchaikov** [comment]
> #### 🔴 CI State: FAILURE
> 
> ✅ - [Build](https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci/14035/) ✅ - [Docker Test](https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci-docker/852/) ✅ - [dtest with tablets](https://jenkins.scylladb.com/job/scylla-master/job/gating-dtest-release-with-tablets/2476/) ✅ - [dtest with topology changes](https://jenkins.scylladb.com/job/scylla-master/job/gating-dtest-release-with-consistent-topology-changes/5593/) ❌ - [dtest](https://jenkins.scylladb.com/job/scylla-master/job/gating-dtest-release/13606/) ❌ - [Offline-installer Artifact Tests](https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci-offline-installer/256/) ✅ - [Unit Tests](https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci/14035/)
> #### Failed Tests (2/36306):
> 
>     * [test_simple_removenode_5](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/14035/testReport/junit/update_cluster_layout_tests/TestUpdateClusterLayout/Tests___dtest___test_simple_removenode_5) [🔍](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_simple_removenode_5)
> 
>     * [test_simple_removenode_5](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/14035/testReport/junit/update_cluster_layout_tests/TestUpdateClusterLayout/Tests___dtest___test_simple_removenode_5_2) [🔍](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_simple_removenode_5)
> 
> 

#20814

**Comment by nyh** [comment]
@tchaikov I didn't wait for the flaky tests to finish before merging this. I took a leap of faith that a spelling correction won't break any real test and merged it before the CI finished :-)

**Pull request closed without merging**
